File: tmp-mei/fgenjs5030319945734616766/base/ControlFlow_1.js
import * as rt from "../rt/main.js";
export class base$$ControlFlow_1 {
  static $self = new base$$ControlFlow_1();

  match$mut(m_m$) {
  return base$$ControlFlow_1.match$mut$fun(m_m$, this);
}

  static match$mut$fun(m_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

}

File: tmp-mei/fgen4255027576711118260/base/ControlFlow_1.java
package base;
public interface ControlFlow_1{
ControlFlow_1 $self = new ControlFlow_1Impl();
Object match$mut(base.ControlFlowMatch_2 m_m$);
static Object match$mut$fun(base.ControlFlowMatch_2 m_m$, base.ControlFlow_1 $this) {
  return (switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (Object) null;
}})
;
}
}